PhotoKeeper/PhotoGridViewController.swift

```diff
@@ -7,114 +7,118 @@
 //
 
 import UIKit
+import RealmSwift
 
 class PhotoGridViewController: UICollectionViewController, UICollectionViewDelegateFlowLayout {
     var store = PhotoStore.shared
-    var images = [Photo]()
+    var photos: Results<PhotoObject>!
```

Comment:
if you see a force unwrap in your code, or anyone else's, it should make you uncomfortable because most crashes in swift happen because of this. Maybe try a computed value here instead:

```swift
var photos: Results<PhotoObject> {
  return store.photos.sorted(byKeyPath: "dateAdded", ascending: false)
}
```